collect_app/src/test/java/org/odk/collect/android/database/DatabaseConnectionTest.kt

```diff
@@ -0,0 +1,39 @@
+package org.odk.collect.android.database
+
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.odk.collect.android.database.forms.FormDatabaseMigrator
+import org.odk.collect.shared.TempFiles.createTempDir
+import java.io.File
+
+@RunWith(AndroidJUnit4::class)
+// https://github.com/getodk/collect/issues/5042
```

Comment:
We should move this down to the individual test so that it doesn't end up being out of context if we add a second test case.

Reply:
Done.